docs/src/administration/users.md

```diff
@@ -3,136 +3,173 @@ title: "User administration"
+---
 
-In this example, using either `platform get` with the CLI:
+1. In the console, select the project where you want to delete a user.
+2. Under **Settings**, click **Access**.
+3. Select the user you want to delete and click **Delete**.
+4. Click **Save**.
 
-```bash
-$ platform get <projectID>
+<--->
 
-```
+---
+title=Using the CLI
+file=none
+highlight=false
+---
+To delete existing users:
 
-or the `git clone` command visible from the "Git" dropdown in the management console
 
 ```bash
-$ git clone git@github.com:user/github-repo.git Project Name
+platform user:delete user1@example.com
 ```
 
-both would error with
+{{< /codetabs >}}
 
-```bash
-Failed to connect to the Git repository: git@github.com:user/github-repo.git
+Once you delete a user, they can no longer access the project.
+After you delete a user from a project or an environment type, you must trigger a redeploy to propagate SSH access changes to each environment.
+You can redeploy by clicking **Redeploy** in the console or  using the CLI command `platform redeploy`.
 
-Please make sure you have the correct access rights and the repository exists.
-```
+### Change existing permissions for environment types
+
+To change user permissions, follow these steps:
 
-despite their **Project Admin** access to the project.
+{{< codetabs >}}
 
-This enhaces consistency over your source code, as the Platform.sh project functions as a read-only mirror of your remote repository. Otherwise, changes pushed directly to the project would be overwritten or deleted when commits are pushed via the integration. Platform.sh considers your integrated remote repository to be the "source of truth" as soon as it has been configured, and this caveat ensures that all commits go through the integration.
+---
+title=In the console
+file=none
+highlight=false
+---
 
-The best course of action is to have your access updated on the integrated repository. If for some reason that is not a quick change, you can still clone through the project using the legacy pattern (which will set the *project* as its remote), but again, it is not recommended that you commit to the project once you have done so:
+1. In the console, select the project where you want to change the user permissions.
+2. Under **Settings**, click **Access**.
+3. Select a user and change the permissions.
+4. Click **Save**.
+
+<--->
+
+---
+title=Using the CLI
+file=none
+highlight=false
+---
+Say you want `user1@example.com` to have the Viewer role for Production environments
+and the Contributor role for Development environments:
 
 ```bash
-$ git clone <project>@git.<region>.platform.sh:<project>.git
+platform user:update user1@example.com -r production:v,development:c
 ```
 
-## Transfer ownership
+After you change a user's role for an environment type, you must trigger a redeploy each environment to propagate access changes. You can redeploy using the CLI command `platform redeploy`.
+
+{{< /codetabs >}}
+
+### Transfer project ownership
```

Comment:
Given the discussion above and the fact that this has as its first step inviting someone to the project, I don't think it falls within the section about managing existing users.

There's a good case to be made that it isn't about managing users at all, but about managing the organization, but I can see that it might make sense here, too.

But also, maybe make it clear that the first step doesn't require inviting a new user, just that the user has to be a Project Admin?